coral-hive/src/test/java/com/linkedin/coral/hive/hive2rel/HiveToRelConverterTest.java

```diff
@@ -62,6 +62,40 @@ public void testBasic() {
     verifyRel(rel, expected);
   }
 
+  @Test
+  public void testBasic2() {
```

Comment:
Can we share the common code between `testBasic` and `testBasic2`?

Reply:
@findepi @losipiuk FYI.

Reply:
Just fixed it by extracting the common code into a separate function.